### ISSUE TITLE
AFL can't determine eligible services

### ISSUE THREAD

**Issue by tigris** [aussieaddons/repo#77]
Seems to get stuck at this dialog every time.

https://github.com/aussieaddons/issue-reports/issues/4645 is my debug report.

I'm using the "free offer" config option. I can watch AFL/NRL on my tablet/phone with the same login credentials no worries, both over mobile data and wifi (the same wifi kodi is trying to use).

**Comment by andybotting** [comment]
Some things you could try:
* Change the subscription type to paid but use your same Telstra creds
* Use option #2 and use the mis-uuid for your auth instead (see http://aussieaddons.com/addons/afl/)

Let me know if either of these help

**Comment by glennguy** [comment]
Hi @tigris

The 'free offer' option will work if you have your mobile account linked to your Telstra ID. You can check this by logging in at telstra.com and seeing if your mobile service, bills etc are viewable. If you've linked your Telstra ID to your activated live pass in the AFL app then change it to the paid/linked option, it should work then.

Let us know how you go :smile:

**Comment by tigris** [comment]
Thanks @glennguy - that explains it. I transferred my mobile billing to my company a few months back, which is when it stopped working.

Changing to the paid option worked, thanks @andybotting 👍 

I tried the `mis-uuid` option earlier today but couldn't get it to work. I noticed the `uuid` was different on my PC to my tablet though. I also wasn't sure if i needed to include the `mis-uuid-` in the input box, or just the hex digest after it.
